src/js/entities/subreddit.test.js

```
@@ -0,0 +1,9 @@
+import subreddit from './subreddit';
+
+describe('Subreddit factory function', () => {
```

You might be missing the case when you create the subreddit and don't pass a name c:

---

True, now wondering if that should even be allowed or if it should throw an exception...

---

It may throw an exception I think

---

It does now, there are some other exceptions missing, and other entities also should throw more stuff but they shoud be added for next pr.